composer.json

```diff
@@ -32,8 +32,9 @@
         "symfony-scripts": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
+            "@php bin/console cache:clear --no-warmup --ansi",
+            "@php bin/console cache:warmup --ansi",
+            "@php bin/console assets:install web --symlink --relative --ansi",
```

Comment:
My point was to stop using extra config and replace them with real cli arguments and options.

Do you mean that this will be a kind of a BC break?